ubyssey/views/main.py

```diff
@@ -210,7 +210,9 @@ def section(self, request, slug=None):
             'type': 'section',
             'featured_articles': {
                 'first': featured_articles[0],
-                'rest': featured_articles[1:4]
+                'row1': featured_articles[1:4],
+                'row2': featured_articles[4:7],
+                'row3': featured_articles[7:10]
```

Comment:
I just discovered this, but Django has a `slice` template tag that would allow you to do this slicing directly in the template: https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#slice